packages/core/src/scss/components/checkout/ratepayForm/_ratepayForm.scss

```diff
@@ -0,0 +1,14 @@
+.dynamic-form-field--ratepayPhoneCountryCode {
+  display: inline-flex;
+  width: 20%;
+}
+
+.dynamic-form-field--ratepayPhoneNumber {
+  display: inline-flex;
+  width: calc(80% - 10px);
```

Comment:
Can we avoid hard coded values here as well please?

Reply:
You mean 10px?